virttest/qemu_devices/qdevices.py

```diff
@@ -1277,6 +1310,52 @@ def verify_unplug(self, out, monitor):
         return not self._is_attached_to_qemu(monitor)
 
 
+class QThrottleGroup(QObject):
+    """
+    throttle-group object.
+    """
+
+    def __init__(self, group_id, props):
+        self.raw_limits = props.copy()
```

Comment:
From the `update_throttle_group`  int patch https://github.com/autotest/tp-qemu/pull/2346, 
I saw you need to set the limits props by the attributes raw_limits
I suggest add methods to do that and set the `raw_limits` to protect attr `_raw_limits`
like:
```
def get_limits_props():
    return self._raw_limits

def update_limits_props(props)
    self._raw_limits.update(props)
```
Keep the attributes of the class object transparent for the caller.  so that we could control the attributes behaviors from the class side which provides a unified interface for the callers.

Reply:
There is another better approach:
```
@property
def raw_limits():
    return self._raw_limits

@raw_limits.setter
def raw_limits(props)
    self._raw_limits.update(props)
```

Reply:
Thanks,updated.